tests/data_context/cloud_data_context/test_expectation_suite_crud.py

```diff
@@ -1,4 +1,5 @@
-from typing import Callable, Tuple
+from collections import namedtuple
```

Comment:
Sorry minor point but could you use the `typing` version?
https://docs.python.org/3/library/typing.html#typing.NamedTuple

Reply:
done!